baybe/objectives/single.py

```diff
@@ -0,0 +1,26 @@
+"""Functionality for single-target objectives."""
+
+import pandas as pd
+from attr import define, field
+from attr.validators import instance_of
+
+from baybe.objectives.base import Objective
+from baybe.targets.base import Target
+
+
+@define(frozen=True)
+class SingleTargetObjective(Objective):
+    """An objective focusing on a single target."""
+
+    _target: Target = field(validator=instance_of(Target))  # type: ignore[type-abstract]
```

Comment:
remind me: attrs is automatically renaming the field name to `target` right? Inever really likes that althoug of course there are prob reasons. Should ew make it explicit in the field?

Reply:
Yes, this is auto-renamed by stripping away the leading underscore. If you want, I can change that (would also fix one open issue in VScode) but then I'd prefer having a mini PR which I can push after merge that does so in all our classes. What do you think?

Reply:
i think changing it now here already doesnt in any way stand in the way of having another PR doing it as well for other parts

Reply:
done